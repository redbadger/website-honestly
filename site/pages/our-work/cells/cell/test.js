import React from 'react';
import { shallow } from 'enzyme';

import Cell from '.';

describe('pages/our-work/cells/cell', () => {
  it('renders a cell with a client image', () => {
    const cellData = {
      clientName: 'Haller',
      clientLogo: '/assets-honestly/haller-38WYH.png',
      image: '/assets-honestly/Hall-3uKBy.jpg',
      headerText: 'Developing technology for good',
      descriptionText:
        'Red Badger teamed up with the Haller Foundation on a pro-bono basis to develop a mobile application which helps Kenyan farmers.',
      routeKey: 'hallerCaseStudy',
    };

    const grid = shallow(<Cell {...cellData} />);
    expect(grid).toMatchSnapshot();
  });

  it('renders a cell without a client image and large header', () => {
    const cellData = {
      clientName: 'Haller',
      clientLogo: '/assets-honestly/haller-38WYH.png',
      headerText: 'Developing technology for good',
      descriptionText:
        'Red Badger teamed up with the Haller Foundation on a pro-bono basis to develop a mobile application which helps Kenyan farmers.',
      routeKey: 'hallerCaseStudy',
    };

    const grid = shallow(<Cell {...cellData} />);
    expect(grid).toMatchSnapshot();
  });
});
