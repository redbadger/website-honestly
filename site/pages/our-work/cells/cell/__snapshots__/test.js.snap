// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/our-work/cells/cell renders a cell with a client image 1`] = `
<div
  className="cell cell-Haller"
>
  <div
    className="caseStudyContentContainer"
  >
    <div
      className="caseStudyContent"
    >
      <Link
        to="hallerCaseStudy"
      >
        <img
          className="clientImage"
          src="/assets-honestly/Hall-3uKBy.jpg"
        />
        <img
          alt="Haller logo"
          className="logo"
          src="/assets-honestly/haller-38WYH.png"
        />
        <h2
          className="normalHeader"
        >
          Developing technology for good
        </h2>
        <p
          className="description"
        >
          Red Badger teamed up with the Haller Foundation on a pro-bono basis to develop a mobile application which helps Kenyan farmers.
        </p>
        <div
          className="links"
        >
          <p
            className="readmore"
          >
            Read more
          </p>
        </div>
      </Link>
    </div>
  </div>
</div>
`;

exports[`pages/our-work/cells/cell renders a cell without a client image and large header 1`] = `
<div
  className="cell cell-Haller"
>
  <div
    className="caseStudyContentContainer"
  >
    <div
      className="caseStudyContent"
    >
      <Link
        to="hallerCaseStudy"
      >
        <img
          alt="Haller logo"
          className="logo"
          src="/assets-honestly/haller-38WYH.png"
        />
        <h2
          className="largeHeader"
        >
          Developing technology for good
        </h2>
        <p
          className="description"
        >
          Red Badger teamed up with the Haller Foundation on a pro-bono basis to develop a mobile application which helps Kenyan farmers.
        </p>
        <div
          className="links"
        >
          <p
            className="readmore"
          >
            Read more
          </p>
        </div>
      </Link>
    </div>
  </div>
</div>
`;
